### ISSUE TITLE
Windows 7: Hangs on health checks [393/394]

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#109]
```
What steps will reproduce the problem?
1. Start test
2. "Running initial health checks on 394 servers (35 threads)... [393/394]
3. Hangs

What is the expected output? What do you see instead?
No output, hangs on above screen. Task manager shows no process activity.

What version of the product are you using? On what operating system?
Downloaded today. Windows 7 Home Premium.


Please provide any additional information below.

```

Original issue reported on code.google.com by `PCSpecia...@gmail.com` on 14 Mar 2010 at 8:20

**Comment by GoogleCodeExporter** [comment]
```
Hmm, this is the second time someone has mentioned this on Windows 7. The other 
person 
was able to re-run the program and make it through successfully. If you re-run 
it, 
does it work properly?

```

Original comment by `helixblue` on 14 Mar 2010 at 10:23
- Changed title: **Windows 7: Hangs on health checks [393/394]**

**Comment by GoogleCodeExporter** [comment]
```
Tried 4 times, same result. Also when I click "X" to close it doesn't close 
until
Windows pops up "program is not responding". I also tried removing 
redownloading and
reinstalling.
```

Original comment by `PCSpecia...@gmail.com` on 14 Mar 2010 at 1:08

**Comment by GoogleCodeExporter** [comment]
```
same result on Vista Home - trying it again with the 'censor check' unchecked...
```

Original comment by `richdepp...@gmail.com` on 18 Mar 2010 at 4:54

**Comment by GoogleCodeExporter** [comment]
```
I also have a second computer (it's also Windows 7) and it hangs at the same 
point
[393/394].  I don't have any Vista or XP computers here to test it on, but I 
suspect
they would hang at the same point.  Maybe somebody who understands what is 
actually
happening at step 393 will have the solution.


```

Original comment by `PCSpecia...@gmail.com` on 18 Mar 2010 at 4:43

**Comment by GoogleCodeExporter** [comment]
```
I get it too. It hangs at different spots for me and sometimes it works fine and
sometimes it freezes up my computer. When I run namebench, I do not have any 
other
application open so it's weird. I'm using 1.1 now and that works fine.
```

Original comment by `Midtown1...@gmail.com` on 22 Mar 2010 at 3:24

**Comment by GoogleCodeExporter** [comment]
```
Happens to me as well. I'm using version 1.2 on Windows 7 64bit.  Same as 
described 
above.
```

Original comment by `latos.a...@gmail.com` on 24 Mar 2010 at 2:44

**Comment by GoogleCodeExporter** [comment]
```
same issue for me 393/394 hangs
Windows 7 64bit (start namebench like administrator)
```

Original comment by `speednpo...@gmail.com` on 23 Apr 2010 at 9:58

**Comment by GoogleCodeExporter** [comment]
```
Can you try the newly released namebench 1.3-BETA1 to see if it resolves this 
bug?
```

Original comment by `helixblue` on 20 May 2010 at 3:24

**Comment by GoogleCodeExporter** [comment]
```
Same issue in Vista home premium....
downloaded beta 1.3 and the problem didn't occur.
program ran to completion and produced results.
```

Original comment by `dickybi...@gmail.com` on 22 May 2010 at 3:38

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3.

```

Original comment by `helixblue` on 31 May 2010 at 6:44
- Changed state: **Fixed**
